src/components/Detail/DetailBackToListButton.jsx

```diff
@@ -0,0 +1,19 @@
+import { useLocation } from 'react-router-dom';
+import styles from './styles/DetailBackToListButton.module.css';
+import { useNavigate } from 'react-router-dom';
+
+export default function DetailBackToListButton() {
+  const navigate = useNavigate();
+  const location = useLocation();
+  const searchParams = new URLSearchParams(location.search);
```

Comment:
💊 제안
search 값만 필요하니 아래처럼 하시는 것이 더 좋을 것 같아요.
또한 선언만 되고 사용되지 않는 searchParams 변수는 제거하시는 것이 좋습니다~

```suggestion
  const navigate = useNavigate();
  const { search } = useLocation();
```